custom/conf/app.example.ini

```
@@ -1470,6 +1470,8 @@ LEVEL = Info
 ;;
 ;; Default configuration for email notifications for users (user configurable). Options: enabled, onmention, disabled
 ;DEFAULT_EMAIL_NOTIFICATIONS = enabled
+; Disabled modules from user settings, could be deletion now, more moudels can be defined in future
```

```suggestion
;; Disabled modules from user settings, could be "deletion" now, more modules can be defined in future
```

Maybe, I think "features" is a better word than "modules". "feature" is a bigger concept than "module".